python/py-buildbot-pkg/Portfile

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-buildbot-pkg
+version             2.3.1
+revision            0
+
+maintainers         nomaintainer
```

Comment:
This is partially a question for @Korusuke & @reneeotten: do we want to encourage developers to put their name here? I mean in the sense that the port would still work without modifications, but upt would leave a comment, and  before submitting the PR users would at least need to delete one comment and deliberately decide for either leaving the port as `nomaintainer`, or for putting their name to the list.

Reply:
This has now been modified.
It is a now a commented line which the maintainer needs to edit. (yet to merge, partially done)